persistence/src/main/java/com/epam/esm/repository/impl/CertificateRepositoryImpl.java

```diff
@@ -81,6 +81,9 @@ INSERT INTO gift_certificate(name, description, price, duration,
     private static final String SQL_INSERT_GENERATED_KEYS = """
             INSERT INTO tags_gift_certificates(tag_id, gift_certificate_id)
             VALUES (?,?)""";
+    private static final String SQL_DELETE_KEYS = """
```

Comment:
Ops.... very strange - use CASCADE or what's the sense of this sql

Reply:
Hmmm, I have cascade, but only on delete operations. And I know nothing about cascade insert.